lib/cadet/assessments/assessments.ex

```diff
@@ -893,10 +997,36 @@ defmodule Cadet.Assessments do
       end)
       |> Repo.transaction()
 
-      Cadet.Accounts.Notifications.handle_unsubmit_notifications(
-        submission.assessment.id,
-        Repo.get(CourseRegistration, submission.student_id)
-      )
+      case submission.student_id do
+        # Team submission, handle notifications for team members
+        nil ->
+          team = Repo.get(Team, submission.team_id)
+
+          query =
```

Comment:
This should be able to be abstracted under `find_team` right?

Reply:
This is a different case. Find team takes in assessment id and cr id as input. Here we can directly use the submission.team_id to retrieve the team. If Line 1003 is executed, it is guaranteed to have a team. Unlike find team, which can return nil.